### PR TITLE
Rework lovelace views page

### PR DESCRIPTION
**Pull request by c727**
**Description:**

- added config for more keys
- simplified examples
- kiss

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by frenck** [comment]
@c727 Build failed?

**Comment by c727** [comment]
it has something to do with the config block. the build system sucks, I need 13min for a build and changes don't pick up. Also the error output is a joke

**Comment by frenck** [comment]
@c727 Known issue... Jekyll 😢 
Recommendation: Setup a local environment (much quicker).

**Comment by c727** [comment]
...that's what I'm talking about

File: source/lovelace/views.markdown [hunk 1]
```diff
@@ -9,88 +9,115 @@ sharing: true
...
+  keys:
+    title:
+      required: true
+      description: A title for this view.
+      type: string
+    cards:
+      required: true
+      description: List of cards to render in this view.
+      type: list
+    id:
```
Comment (sjabby) [inline source/lovelace/views.markdown]: I believe this needs to be "id" to validate with netlify.

Comment (c727) [reply source/lovelace/views.markdown]: :church:

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
👍

**Pull request merged**